maxspeed_signals.mss

```diff
@@ -18,10 +18,13 @@
     }
     
     /* German speed signals (Zs 3v) as signs */
-    ["feature"="DE-ESO:zs3v"]["signal_speed_limit_distant_form"="sign"]["signal_speed_limit_distant_speed"=null] {
-      marker-file: url('symbols/de/zs3v-empty-sign-down.svg');
-    }
     ["feature"="DE-ESO:zs3v"]["signal_speed_limit_distant_form"="sign"] {
+      ["signal_speed_limit_distant_speed"=null] {
+        marker-file: url('symbols/de/zs3v-empty-sign-down.svg');
+        marker-width: 22;
+        marker-height: 19;
+        marker-allow-overlap: true;
```

Comment:
Just out of curiosity: how did it happen that this has a different tag order than the clause below?

Reply:
But thinking again it may indeed be better if you add a new null clause to the block just below to set the image properties, and then a new block just below that that has the same null clause and only adds the icon. This way all settings will be consistent.

Reply:
I just didnt copy...